include/nbl/video/IPhysicalDevice.h

```diff
@@ -362,8 +362,8 @@ class NBL_API2 IPhysicalDevice : public core::Interface, public core::Unmovable
                 storageImage(usages.hasFlags(asset::IImage::EUF_STORAGE_BIT)),
                 transferSrc(usages.hasFlags(asset::IImage::EUF_TRANSFER_SRC_BIT)),
                 transferDst(usages.hasFlags(asset::IImage::EUF_TRANSFER_DST_BIT)),
-                attachment((usages& core::bitflag<asset::IImage::E_USAGE_FLAGS>(asset::IImage::EUF_COLOR_ATTACHMENT_BIT | asset::IImage::EUF_DEPTH_STENCIL_ATTACHMENT_BIT)).value != 0),
-                attachmentBlend((usages& core::bitflag<asset::IImage::E_USAGE_FLAGS>(asset::IImage::EUF_COLOR_ATTACHMENT_BIT)).value != 0),
+                attachment(usages.hasFlags(core::bitflag(asset::IImage::EUF_COLOR_ATTACHMENT_BIT)|asset::IImage::EUF_DEPTH_STENCIL_ATTACHMENT_BIT)),
```

Comment:
A bit odd if a format both supports color attachment and depth stencil at the same time? 
@deprilula28 are you sure we don't want OR instead of AND when setting this attachment.
I will touch these soon so we don't need to fix right now.

Reply:
aaah `hasFlags` is a AND + Compare, not an AND !=0

sorry my bad, please revert